src/common/header/HeaderNav.jsx

```diff
@@ -88,15 +88,15 @@ const HeaderNav = ({ showBrowse }) => {
             </li>
           )}
           <li>
-            <Link
+            <a
```

Comment:
@Angra974 , Can you try this?

```suggestion
            <Link
              className="app-header-btn app-header-btn--secondary"
              data-testid="events-btn"
              target="_blank"
              to={pathname:"https://hustles.reactplay.io"}
            >
```

Reply:
Same result.
![image](https://github.com/reactplay/react-play/assets/48807534/e1d4207f-fab4-4d37-a326-90cfed2efb13)

Reply:
> @Angra974 , Can you try this?

a should come inside Link, isn't it?

Reply:
@Angra974 , did you try the same suggestion?

Reply:
> hname:"https://hustles.reactplay.io"}}

Yes, it's ok with your code.

Reply:
I swap my browser to see in another one.

Reply:
No. Didn't work. Try it in Chrome but got the localhost before again. 
![image](https://github.com/reactplay/react-play/assets/48807534/bf92a11e-d565-4f65-ba76-4d18fcb8ad48)